app/src/main/res/drawable/progress_bar.xml

```diff
@@ -2,21 +2,26 @@
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:id="@android:id/background">
         <shape>
-            <solid android:color="?attr/progressBackground" />
+            <corners android:radius="8dp"/>
+            <solid android:color="?attr/progressBackground"/>
         </shape>
     </item>
-    <item android:id="@android:id/secondaryProgress">
-        <clip>
+    <item android:id="@android:id/secondaryProgress" >
+
+        <scale android:scaleWidth="100%">
             <shape>
-                <solid android:color="?attr/progressSecondary" />
+                <corners android:radius="8dp"/>
+                <solid android:color="?attr/progressSecondary"/>
             </shape>
-        </clip>
+        </scale>
     </item>
-    <item android:id="@android:id/progress">
-        <clip>
+    <item android:id="@android:id/progress" >
+
```

Comment:
You can remove this empty line and the other one (line 10) and reformat the file.